ansible/roles/actors/tasks/main.yml

```diff
@@ -21,6 +21,17 @@
   with_items: "{{ actors.packages }}"
   when: ansible_distribution == 'CentOS'
 
+- name: "Install Leapp-Actors dependencies"
+  become: true
+  shell: "cd {{ actors.git.dest }} && make install-deps"
```

Comment:
This seems to work, but maybe replace the shell module for the make one. It is flagged as "preview" but seems to work on my tests. Would be something like this:
```
make:
  chdir: {{ actors.git.dest }}
  target: install-deps
``` 
Maybe even place the targets inside a for each loop

Reply:
I am not sure whether we really want to do this. Did you test it in Centos as well?